pre-commit-config.yaml

```diff
@@ -24,63 +26,50 @@ repos:
     - id: trailing-whitespace
     - id: mixed-line-ending
       args: ['--fix=lf']
-    - id: flake8
```

Comment:
`flake8` is not part of pre-commit-hooks repository anymore. We could check where it was moved if we want.

Reply:
I don't have any strong opinions about flake8 directly, but there is probably benefit in fewer tools here. The largest point of friction in our linting as so far been tools with competing changes.

It doesn't seem like we'd lose anything major by dropping flake8. Is there any functionality you'd be concerned about losing?

Reply:
Nothing really important. I'm not sure how black behaves on some cases, but for `flake8-deprecated` should trigger an error if you use things that are deprecated in Python where I assume that black, would just format it correctly and say nothing about it.

Mainly, flake8 plugins are small nice things but are not essential. So, I'm fine removing them for now instead of investing time trying to keep them. At least, at this point.

Besides, we weren't really running these checks --so, we won't loose anything from that perspective 😄

Reply:
Yeah, that was my understanding as well. We'd lose some of these with just running black, but most of the flake8 checks seem low priority.